mozilla/fireplace
# feed UA tracking events (bug 1049280)

[So many events](https://docs.google.com/document/d/1sC2Bp65v3UW-mcsqPcIM23D7XxeOt_QB2CRbkZ-FDb4/edit#)

To test:
- Click around.
- Check that everything in `require('tracking_events').track_log` checks out.
